cesium_app/tests/fixtures.py

```diff
@@ -28,9 +28,15 @@ def create_test_project():
 
 
 @contextmanager
-def create_test_dataset(project):
-    header = pjoin(os.path.dirname(__file__), 'data', 'asas_training_subset_classes.dat')
-    tarball = pjoin(os.path.dirname(__file__), 'data', 'asas_training_subset.tar.gz')
+def create_test_dataset(project, type='class'):
+    if type == 'class':
```

Comment:
Probably best not to use type, which is a Python built-in.  Also, can you add a little docstring here to say what "type" means?